### PR TITLE
Docs on generators actions linting

### PR DESCRIPTION
**Pull request by JackHowa**
# Goals

- [x] Document and audit all existing automation in a readme in plain dev language
- [x] Document hygen generation and available commands
- [x] Document how github actions work with lerna
- [x] Discuss storybook

### CODE REVIEW

File: .storybook/manager-head.html [hunk 1]
```diff
@@ -1,44 +1 @@
 <!-- reference for customizing https://github.com/storybookjs/storybook/issues/11222#issuecomment-646411961 -->
-<link rel="stylesheet" href="https://nav-prod.home.static.arcpublishing.com/css/arcnav.css" />
```
Comment (matthewroach) [inline .storybook/manager-head.html]: There are no current plans to not render this in ALC going forward - while that could change, we should keep this until we decided either way

Comment (JackHowa) [reply .storybook/manager-head.html]: ok will revert

Comment (JackHowa) [reply .storybook/manager-head.html]: updated

**Review by matthewroach — APPROVED** [review]
Thanks for the clean up @JackHowa

**Pull request merged**